pkgs/os-specific/linux/nvidia-x11/open.nix

```diff
@@ -0,0 +1,37 @@
+nvidia_x11: sha256:
+{ stdenv
+, lib
+, fetchFromGitHub
+, kernel
+}:
+
+stdenv.mkDerivation {
+  pname = "nvidia-open";
+  version = "${kernel.version}-${nvidia_x11.version}";
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "open-gpu-kernel-modules";
+    rev = nvidia_x11.version;
+    inherit sha256;
```

Comment:
nit: we're currently trying to move everything towards SRI hashes and off of the sha256 attribute

Reply:
Sadly, the userland tools remain proprietary as userland drivers.